3기/LeeGaYeon/week2_mission/css/style.css

```
@@ -0,0 +1,319 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: 'Inter';
+  src: url(https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css) format(('woff'));
+}
+
+@font-face {
+  font-family: 'Inter';
+  src: url('https://fonts.googleapis.com/css2?family=Inter&display=swap') format('woff');
+  unicode-range: U+0041-005A, U+0061-007A;
+}
+
+h2 {
+  color: #2d3436;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter';
+  font-size: 16px;
+  line-height: 160%;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#wrap {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  width: 100%;
+  height: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 120px;
+  min-height: 100px;
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+}
+
+h1 {
+  font-size: 20px;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100px;
+  background-color: #2d3436;
+}
+
+.footer-content {
+  color: #dfe4ea;
+}
+
+main {
+  flex: 1;
+  display: flex;
+}
+
+article.about {
+  flex: 1;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 120px;
+}
+
+article.about::before {
+  content: '';
+  background-image: url('../images/background-img.png');
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  opacity: 0.1;
+  top: 0;
+  left: 0;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.introduce {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.introduce h2,
+.introduce p {
+  margin: 0;
+}
+
+.skills {
+  display: flex;
+  gap: 16px;
+}
+
+article.timeline {
+  background-color: #fafafa;
+  flex: 1;
+  padding: 100px 120px;
+}
+
+.timeline-p {
+  color: #ff7f50;
+  margin: 0;
+}
+
+.timeline-title h2 {
+  order: 1;
+  margin: 0;
+  font-size: 28px;
+}
+
+.timeline-title {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  width: 100%;
+}
+
+.timeline-content {
+  width: 100%;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  padding: 0;
+}
+
+.content {
+  display: grid;
+  font-size: 16px;
+  grid-template-columns: 1fr 2fr;
+}
+
+.period {
+  color: #2d3436;
+  font-weight: 700;
+  font-size: 18px;
+  min-width: 150px;
+  display: flex;
+  align-items: center;
+}
+
+.main-content {
+  border-left: 1px solid;
+  border-color: #636e72;
+  padding: 0 16px;
+}
+
+.main-content > h2,
+p {
+  margin: 0;
+  font-size: 16px;
+}
+
+.desc {
+  color: #636e72;
+}
+
+nav {
+  display: flex;
+  gap: 40px;
+}
+
+button {
+  border: none;
+  background-color: #fff;
+  font-size: 14px;
+}
+
+strong {
+  color: coral;
+  font-weight: 900;
+}
+
+button {
+  cursor: pointer;
+}
+
+button:active {
+  font-weight: 700;
+}
+
+button.active {
+  font-weight: 700;
+}
```

두 속성 모두 active를 나타낸다면,

```suggestion
button:active,
button.active {
  font-weight: 700;
}
```

이렇게 한번에 묶어 쓰는것도 방법이 되어요.